x-pack/plugins/security_solution/public/resolver/view/assets.tsx

```diff
@@ -475,7 +475,7 @@ export const useResolverTheme = (): {
       ),
       isLabelFilled: false,
       labelButtonFill: 'primary',
-      strokeColor: `${theme.euiColorPrimary}33`, // 33 = 20% opacity
```

Comment:
Maintaining the same outline color since the more translucent outline now signifies the analyzed event